test/e2e/utils/record-helpers.js

```diff
@@ -58,6 +58,8 @@ exports.testPresentation = function (tableParams) {
 	it("should render columns which are specified to be visible and in order", function() {
 		var columns = tableParams.columns.filter(function(c) {return c.value != null;});
 		chaisePage.recordPage.getAllColumnCaptions().then(function(pageColumns) {
+			browser.sleep(2000);
```

Comment:
This sleep shouldn't be here and doesn't appear to be making a difference in the tests either.